misc/systemd/kubo-api.socket

```diff
@@ -2,11 +2,11 @@
 # config.
 
 [Unit]
-Description=IPFS API Socket
+Description=Kubo's API Socket
 
 [Socket]
-Service=ipfs.service
-FileDescriptorName=io.ipfs.api
+Service=kubo.service
+FileDescriptorName=io.kubo.api
```

Comment:
Not sure if this matters, but this reverse DNS notation is problematic – we don't own `kubo.io` :grimacing:

Reply:
Ah okay, I thought the blank webpage would be a placeholder :)